analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RemoveObsoleteCode.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Tests
```

Comment:
Namespace does not anything here.

Reply:
They allow specifying the `// ^^^^` maker for the class attribute. I will therefore leave it.